test/Window/Keyboard.test.cpp

```diff
@@ -0,0 +1,60 @@
+#include <SFML/Window/Keyboard.hpp>
+
+// Other 1st party headers
+#include <SFML/System/String.hpp>
+
+#include <catch2/catch_test_macros.hpp>
+
+#include <WindowUtil.hpp>
+
+#include <cctype>
+
+TEST_CASE("[Window] sf::Keyboard", runDisplayTests())
+{
+    SECTION("isKeyPressed(Key)")
+    {
+        CHECK(!sf::Keyboard::isKeyPressed(sf::Keyboard::W));
+        CHECK(!sf::Keyboard::isKeyPressed(sf::Keyboard::A));
+        CHECK(!sf::Keyboard::isKeyPressed(sf::Keyboard::S));
+        CHECK(!sf::Keyboard::isKeyPressed(sf::Keyboard::D));
+    }
+
+    SECTION("isKeyPressed(Scancode)")
+    {
+        CHECK(!sf::Keyboard::isKeyPressed(sf::Keyboard::Scan::W));
+        CHECK(!sf::Keyboard::isKeyPressed(sf::Keyboard::Scan::A));
+        CHECK(!sf::Keyboard::isKeyPressed(sf::Keyboard::Scan::S));
+        CHECK(!sf::Keyboard::isKeyPressed(sf::Keyboard::Scan::D));
+    }
+
+    SECTION("localize(Scancode)")
+    {
+        CHECK(sf::Keyboard::localize(sf::Keyboard::Scan::Space) == sf::Keyboard::Space);
+    }
+
+    SECTION("delocalize(Key)")
+    {
+        CHECK(sf::Keyboard::delocalize(sf::Keyboard::Space) == sf::Keyboard::Scan::Space);
+    }
+
+    SECTION("getDescription(Scancode)")
+    {
+        CHECK(std::toupper(static_cast<int>(sf::Keyboard::getDescription(sf::Keyboard::Scan::W)[0])) == 'W');
+        CHECK(std::toupper(static_cast<int>(sf::Keyboard::getDescription(sf::Keyboard::Scan::A)[0])) == 'A');
```

Comment:
This fails with AZERTY keyboard.

Reply:
I also switched to using the spacebar here